public/app/percona/dbaas/components/DBCluster/DBCluster.hooks.ts

```diff
@@ -16,7 +22,9 @@ const OPERATORS: Partial<OperatorDatabasesMap> = {
   [Databases.mongodb]: Operators.psmdb,
 };
 
-export const useDBClusters = (kubernetes: Kubernetes[]): [DBCluster[], GetDBClustersAction, boolean] => {
+export const useDBClusters = (
+  kubernetes: Kubernetes[]
+): [DBCluster[], GetDBClustersAction, SetDBClustersLoadingAction, boolean] => {
```

Comment:
Let's create an alias for that return type (also in the mock)

Reply:
Done @nicolalamacchia !